src/internet-analyzer/azext_internet_analyzer/custom.py

```diff
@@ -9,39 +9,39 @@
 # pylint: disable=unused-argument
 
 
-def create_internet_analyzer_profile(cmd, client,
-                                     resource_group,
-                                     name,
-                                     location=None,
-                                     tags=None,
-                                     enabled_state=None):
+def begin_create_internet_analyzer_profile(cmd, client,
```

Comment:
Actually you don't need change the name of custom method you defined, and you just need change `client.create_or_update` to `client.begin_create_or_update`.

Reply:
also with these changes do you mind if I keep it as is to maintain the same function name across the sdk and custom.py? I realize it isn't necessary, but just trying to be consistent.

Also, I am assuming what you are talking about also applies to the other functions I have adjust as well?

Reply:
From design perspective, I suggest you keep the name of custom method unchanged as we want wrap sdk breaking change just in implementation level but not method interface level for CLI.

Reply:
adjusted for all custom methods